app/src/app/routes/Sources.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import { Container, Grid, makeStyles } from "@material-ui/core";
+
+import SourceCreate from "features/sources/SourceCreate";
+import SourceForm from "features/sources/SourceForm";
+import SourceGrid from "features/sources/SourceGrid";
+
+const useStyles = makeStyles((theme) => ({
+  gridContainer: {
+    flexGrow: 1,
+    paddingTop: theme.spacing(5),
+  },
+}));
+
+const Sources = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Container maxWidth="xl">
+      <Grid className={classes.gridContainer} spacing={3} container>
+        <Grid item>
```

Comment:
Why do you use Grid container to set only one child inside? Is it only for style purposes?

Reply:
I don't know, I copy/pasted this part. Should I remove this ?